prow/cmd/sub/main.go

```diff
@@ -174,7 +174,7 @@ func main() {
 	pullServer := subscriber.NewPullServer(s)
 	interrupts.Run(func(ctx context.Context) {
 		if err := pullServer.Run(ctx); err != nil {
-			logrus.WithError(err).Error("Failed to run Pull Server")
+			logrus.WithError(err).Fatal("Failed to run Pull Server")
```

Comment:
This probably isn't critical for sub, but we should technically be using `interrupts.Terminate()` to end the process so that other go routines using the `interrupts` package can exit gracefully.
https://github.com/kubernetes/test-infra/blob/4290b6519652a181e6dd32b387140f459274642f/prow/interrupts/interrupts.go#L78-L82